### PR TITLE
doc(instance): use image name instead of static image id

### PR DESCRIPTION
**Pull request by remyleone**

### CODE REVIEW

File: website/docs/index.html.markdown [hunk 1]
```diff
@@ -63,9 +63,14 @@ resource "scaleway_instance_security_group" "www" {
   }
 }
 
+data "scaleway_image" "ubuntu" {
```
Comment (jerome-quere) [inline website/docs/index.html.markdown]: This data source will soon be deprecated

File: website/docs/index.html.markdown [hunk 2]
```diff
@@ -63,9 +63,14 @@ resource "scaleway_instance_security_group" "www" {
...
+    architecture = "x86_64"
+    name = "Ubuntu Bionic"
+}
+
 resource "scaleway_instance_server" "web" {
   type = "DEV1-L"
-  image = "f974feac-abae-4365-b988-8ec7d1cec10d"
+  image = "${data.scaleway_image.ubuntu.id}"
```
Comment (jerome-quere) [inline website/docs/index.html.markdown]: You can directly write `ubuntu` here

Comment (kindermoumoute) [reply website/docs/index.html.markdown]: `ubuntu-bionic`

**Comment by remyleone** [comment]
Fixed

**Pull request merged**